include/slb/type_traits.hpp

```diff
@@ -903,11 +903,63 @@ using add_pointer_t = typename slb::add_pointer<T>::type;
 // 23.15.7.6, other transformations
 
 using std::aligned_storage;
-using std::aligned_union;
 using std::decay;
 using std::enable_if;
 using std::conditional;
 
+#if !defined(__GLIBCXX__) || __has_include(<codecvt>) // >= libstdc++-5
+// libc++ does not provide an out-of-class definition for `alignment_value`.
+#if !defined(_LIBCPP_VERSION)
+#define SLB_CPP_LIB_ALIGNED_UNION 2 // available / conforming
+#else
+#define SLB_CPP_LIB_ALIGNED_UNION 1 // available / non-conforming
+#endif
+#else
+#define SLB_CPP_LIB_ALIGNED_UNION 0 // not available
+#endif
+
+#if SLB_CPP_LIB_ALIGNED_UNION == 2
+using std::aligned_union;
+#else
+#if SLB_CPP_LIB_ALIGNED_UNION == 1
+template <std::size_t Len, typename... Types>
+struct aligned_union : std::aligned_union<Len, Types...> {
+  static constexpr std::size_t alignment_value =
+      std::aligned_union<Len, Types...>::alignment_value;
+};
+#else
+namespace detail {
+template <std::size_t... Vs>
+struct static_max {
+  static constexpr std::size_t value = 0u;
+};
+
+template <std::size_t V0>
+struct static_max<V0> {
```

Comment:
I think calling this `V0` is confusing seeing as it's not the same `V0` from the recursive case.
Can we just call it `V`?